docs/cli/kops_rolling-update_cluster.md

```diff
@@ -75,7 +75,7 @@ kops rolling-update cluster [flags]
       --force                          Force rolling update, even if no changes
   -h, --help                           help for cluster
       --instance-group strings         List of instance groups to update (defaults to all if not specified)
-      --instance-group-roles strings   If specified, only instance groups of the specified role will be updated (e.g. Master,Node,Bastion)
+      --instance-group-roles strings   If specified, only instance groups of the specified role will be updated (e.g. Master,Apiserver,Node,Bastion)
```

Comment:
Why do we write "Apiserver" for role name for _kops rolling-update cluster_, but write "APIServer" for the role name for _kops create instancegroup_?